<html>
  
<p><h2>RESEARCH INTERESTS</h2></p>

<p>Investigations of computational methods and the development of software technologies to advance the scientific understanding of genomic data for application in the healthcare industry.
</p>

<p><h2>EDUCATION</h2></p>

<p><b>Master of Science student | Aug 2018 – Present</b><br>
Bioinformatics & Computational Biology <br>
  <em>Saint Louis University, St. Louis, MO.</em></p>
  <ul style="list-style-type:disc;margin-top:0;">
  <li> GPA: 4.0 </li></ul>

<p style="margin-bottom:0;"><b>Doctor of Philosophy student | Sep 2017 – Apr 2018</b><br>
Ecology, Evolution & Marine Biology Department <br>
  <em>University of California - Santa Barbara, Goleta, CA.</em> </p> 
  <ul style="list-style-type:disc;margin-top:0;">
  <li> GPA: 3.88 </li></ul>

<p style="margin-bottom:0;"><b>Bachelor of Science, Marine Science/Biology | May 2016</b><br>
  <em>University of Miami, Coral Gables, FL.</em></p> <ul style="list-style-type:disc;margin-top:0;">
  <li> Minor, Chemistry</li>
  <li> GPA: 3.669</li></ul>

<p><h2>RESEARCH EXPERIENCE</h2></p>

<p style="margin-bottom:0;"><b>BioHPC Laboratory | Aug 2018 – present</b><br>
  <em>Computer Science Department, St. Louis University</em> </p> <ul style="list-style-type:disc;margin-top:0;">
  <li>Currently working with Dr. Tae-Hyuk Ahn (Computer Science) and Dr. Zhenguo Lin (Biology) to develop an R software package that can more effectively analyze CAGE-sequencing data; the goal of the development is to provide better transcription start site profiles from CAGE-seq datasets.</li>
  <li>Responsible for writing new data analysis functions (e.g. peak calling, clustering, etc.) and for editing and adapting existing functions written by Dr. Lin's graduate student.</li></ul>

<p style="margin-bottom:0;"><b>Proulx Laboratory | Sep 2017 – Apr 2018</b><br>
  <em>EEMB Department, University of California at Santa Barbara</em> </p>
<ul style="list-style-type:disc;margin-top:0;">
  <li>Focused on developing theoretical models to understand fitness and selection processes involved in evolution.</li>
  <li>Assisted with microbiology procedures involved in the experimental evolution studies of the yeast species <i>Saccharomyces cerevisiae</i>.</li></ul>

<p style="margin-bottom:0;"><b>Marine Genomics Laboratory | Jan 2016 – Aug 2016</b><br>
  <em>Department of Marine Biology and Ecology, University of Miami</em> </p>
<ul style="list-style-type:disc;margin-top:0;">
  <li> Collaborated with Atlantic Oceanographic & Meteorological Laboratory of NOAA to investigate the roll of salinity fluctuations in causing selection by analyzing the genetic variation and connectivity among populations of rainwater killifish (<i>Lucania parva</i>) in Florida’s Biscayne Bay; evidence from investigations has the potential to justify the Comprehensive Everglades Restoration Plan, a government initiative spending $10.5 billion over 35 years to restore the Everglades’ natural drainage system. </li>
<li> Investigated connectivity of over-fished populations of parrotfish species in the Bahama islands by determining the potential identity of larval DNA in the parrotfish genus Sparisoma through sequence alignment comparisons and phylogenetic analyses; research findings have the potential to enhance fishing regulations and management strategies in the Bahama islands. </li></ul>

<p><h2>TEACHING EXPERIENCE</h2></p>

<p style ="margin-bottom:0;"><b>Academic Tutor | Oct 2013 – Sep 2016</b><br>
  <em>Prepworks Tutoring</em></p>
  <ul style="list-style-type:disc;margin-top:0;">
  <li>Taught students PSAT, SAT, ACT, Biology, Chemistry and all Mathematics classes in one-to-one and small group environments with average student improvements over 300 SAT points, 5 ACT points, and 2 letter grades. </li>
  <li>Worked approximately 15 hours per week during my undergraduate studies. </li></ul>

<p><h2>HONORS AND AWARDS</h2></p>

<p style ="margin-bottom:0;"><b>Graduate Assistantship | Fall 2018 – Fall 2019</b></p>
<ul style="list-style-type:disc;margin-top:0;">
  <li>Awarded full tuition coverage and a $22,000 annual stipend.</li>
  <li>Required to complete 20 hours of research per week with faculty associated with the Bioinformatics & Computational Biology M.S. program. </li></ul>
  
<p style ="margin-bottom:0;"><b>IGERT Traineeship	| Fall 2017 – Spring 2018</b></p>
<ul style="list-style-type:disc;margin-top:0;">
  <li>An interdisciplinary graduate program in Network Science and Big Data funded by the National Science Foundation and the University of California at Santa Barbara which provides recipients full tuition coverage and a $30,000 annual stipend.</li>
  <li>Requires participating graduate students to take Network Science courses offered in the UCSB Computer Science department.</li>
  <li>Offers graduate students opportunities to conduct interdisciplinary research through laboratory rotations in seven participating departments: Computer Science (CS), Communication, Ecology, Evolution & Marine Biology (EEMB), Electrical & Computer Engineering (ECE), Geography, Mechanical Engineering (ME), and Sociology.</li></ul>

<p style ="margin-bottom:0;"><b>University/President’s Scholarship | 2012 – 2016</b></p>
<ul style="list-style-type:disc;margin-top:0;">
  <li>Covered two-thirds of undergraduate tuition and fees for demonstrating significant academic achievement.</li>
  </ul>

<p style ="margin-bottom:0;"><b>Miami Commitment Program | 2012 – 2016</b></p>
<ul style="list-style-type:disc;margin-top:0;">
  <li>An award-winning, experiential learning program available by invitation to incoming freshman students receiving a University of Miami merit/academic scholarship.</li>
  </ul>	

<p><h2>PROFESSIONAL EXPERIENCE</h2></p>

<p style ="margin-bottom:0;"><b>Bioinformatics Intern | Apr 2018 – Aug 2018</b><br>
  <em>Meyers Laboratory, Donald Danforth Plant Science Center</em></p>
<ul style="list-style-type:disc;margin-top:0;">
  <li>Participated in a paid summer internship in Dr. Blake Meyers lab; the lab focuses on genome-scale studies of small RNAs and their targets in plants.</li>
  <li>Analyzed sequence variation data from the 1001 Genomes Project to identify and visualize intraspecific microRNA variation in 1135 accessions of Arabidopsis; my research investigations found the presence of SNPs affects the stability of miRNA secondary structures and potentially inhibits the regulatory functions of miRNAs in this plant species. </li> 
  <li>Developed bioinformatics pipeline to be used for similar miRNA analyses; specific plans to utilize pipeline to analyze data from the 3000 Rice Genomes Project.</li></ul>	

<p style ="margin-bottom:0;"><b>Web Content Engineer | May 2016 – Oct 2017</b><br>
  <em>Prepworks E-Learning Systems</em></p>
<ul style="list-style-type:disc;margin-top:0;">
  <li>Edited, approved, and published question items, remediation, interactive activities, and videos for SAT, ACT, Algebra, and Geometry courses to improve student comprehension and academic success.</li>
  <li>Created and/or edited approximately 80% of full remediated questions for Prepworks’ Award Winning Adaptive Geometry Course.</li>
  <li>Directed a team of content developers and content digitizers to update and expand content found in Prepworks’ Award Winning Adaptive ACT and SAT courses.</li>
  </ul>	

<p><h2>CAMPUS INVOLVEMENT</h2></p>

<p style ="margin-bottom:0;"><b>Water Polo Club | 2012 – 2016</b><br>
  <em>University of Miami</em></p>
  <ul style="list-style-type:disc;margin-top:0;">
  <li>Team CAPTAIN during 2013 Florida State Championship season.</li>
  <li>PRESIDENT of club during 2014 season.</li></ul>

<p><b>Scuba Club | 2012 – 2016</b><br>
  <em>University of Miami</em></p>

<p><b>Alumnus of Pi Kappa Alpha Fraternity</b><br>				
  <em>University of Miami Chapter</em></p>

<p><h2>RELEVANT SKILLS</h2></p>

<ul style="list-style-type:disc;">
<li>Lab work: Buffer synthesis, restriction enzyme digest, polymerase chain reaction (PCR), genomic DNA isolation, RNA isolation, gel electrophoresis (DNA, RNA), spectrophotometry, DNA barcoding, genotyping by sequencing (GBS), restriction site associated DNA sequencing (RAD-seq), single colony isolation (streaking), optical microscopy (inverted compound microscope)</li>
<li>Software: GenomeTools, bedtools, ViennaRNA Package, Trimmomatic, FastQC, Burrows-Wheeler Aligner (BWA), Anaconda, RStudio, Git, BLAST, Gephi, MEGA, Tassel, Arlequin, PGDSpider, Structure, PAUP, ArcGIS, Adobe Photoshop, Adobe Illustrator, Microsoft Office, Salesforce CRM</li>
  <li>Programming/markup language: Python, R, Unix Shell, C++ (proficient), Java (proficient), HTML/CSS, LaTeX</li></ul>

<p><h2>LANGUAGES</h2></p>

<p>English: Fluent<br>
Latin: Proficient</p>

<p><h2>REFERENCES</h2></p>

<p><b>Blake Meyers, Ph.D., Principal Investigator</b><br>
Small RNA Laboratory<br>
Donald Danforth Plant Science Center<br>
Division of Plant Sciences, University of Missouri – Columbia<br>
(314) 587 - 1422, BMeyers@danforthcenter.org</p>

<p><b>Douglas L. Crawford, Ph.D., Professor</b><br>
Marine Genomics Laboratory<br>
Department of Marine Biology and Ecology<br>
University of Miami, Rosenstiel School of Marine and Atmospheric Science<br>
(305) 421 - 4121, dcrawford@rsmas.miami.edu </p>

<p><b>Marjorie F. Oleksiak, Ph.D., Associate Professor</b><br>
Marine Genomics Laboratory<br>
Department of Marine Biology and Ecology<br>
University of Miami, Rosenstiel School of Marine and Atmospheric Science<br>
(305) 421 - 4341, moleksiak@rsmas.miami.edu</p>

<p><b>Michael Penney, Global Head of Technology</b><br>
Coaches Training Institute (CTI), San Francisco Bay Area<br>
(941) 374 – 0520, michaelbtw@gmail.com </p>

<p><b>Patrick Doherty, founder and CEO</b><br>
Saturday Capital, St. Louis, MO.<br>
(636) 489 – 8522, Patrick.Doherty@SaturdayCapital.com </p>

</html>

